src/bin/travis/resources/centaur.wdl

```diff
@@ -21,7 +33,7 @@ task centaur {
     }
 
     runtime {
-        docker: "geoffjentry/centaur-cromwell:latest" 
+        docker: "us.gcr.io/broad-dsde-cromwell-dev/centaur:latest" 
```

Comment:
I think you've done at least a significant piece of [this ticket](https://github.com/broadinstitute/cromwell/issues/1296)